Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi8/ubi-minimal:8.6 AS builder
+
+RUN microdnf install -y tar gzip make which
+ENV ARCHITECTURE=x64
+# install go 1.17.8
+RUN curl -O -J https://dl.google.com/go/go1.18.4.linux-amd64.tar.gz
```

Comment:
do we really want to go go 1.18? As go 1.20 is out, 1.18 shouldn't get all the updates anymore, right? As per the go[ release policy](https://go.dev/doc/devel/release#policy):

> Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).

Reply:
it depends on the supported version by registry.access.redhat.com/ubi8/ubi-minimal:8.6, I think this can be updated